client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Box from "@material-ui/core/Box";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+// import Grid from '@material-ui/core/Grid';
+import Button from "@material-ui/core/Button";
+// import IconButton from '@material-ui/core/IconButton';
+import AddCircleIcon from "@material-ui/icons/AddCircle";
+import Avatar from "@material-ui/core/Avatar";
+import { makeStyles } from "@material-ui/core/styles";
+import { Link } from "react-router-dom";
+
+const useStyles = makeStyles(() => ({
+  // TODO: mobile scaling
+  // TODO: finalize styles (font, color)
+  // TODO: add logo?
+  root: {
+    flexGrow: 1,
+  },
+  toolbar: {
+    paddingLeft: 20,
+    paddingRight: 15,
+  },
+  button: {
+    minWidth: 100,
+    justifyContent: "left",
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    color: "white",
+    textTransform: "none",
+  },
+  createButton: {
+    margin: 5,
+  },
+  profileButton: {
+    maxWidth: "12%", // to catch long profile name
+  },
+  avatar: {
+    width: "1.5em",
+    height: "1.5em",
+  },
+  title: {
+    flexGrow: 1,
+    color: "white",
+    textDecoration: "none",
+  },
+}));
+
+const PROFILE_PATH = "/profile";
```

Comment:
Sidenote: I edited this to be "/user" in a different commit, but I'll take care of that when it comes time to merge.